scripts/Jenkinsfile

```
@@ -27,7 +26,8 @@ pipeline {
 
         stage('빌드') {
             steps {
-                sh 'cp $spreadsheet_json ./module-infrastructure/google-spreadsheet/src/main/resources/'
+                withCredentials([file(credentialsId: 'GOOGLE_CREDENTIAL_JSON', variable: 'spreadsheetJson')])
+                sh 'cp $spreadsheetJson ./module-infrastructure/google-spreadsheet/src/main/resources/'
```

```suggestion
                withCredentials([file(credentialsId: 'GOOGLE_CREDENTIAL_JSON', variable: 'spreadsheetJson')]) {
                    sh 'cp $spreadsheetJson ./module-infrastructure/google-spreadsheet/src/main/resources/'
                }
```
withCredentials step must be called with a body 오류 해결하려면 이렇게 해야 하는 것 같습니다!